lib/Containers/MerkleTree.cpp

```diff
@@ -57,7 +57,7 @@
 #endif
 
 // can turn this on for more aggressive and expensive consistency checks
-// #define PARANOID_TREE_CHECKS
+#define PARANOID_TREE_CHECKS
```

Comment:
I suppose we don't want this to be enabled by default
```suggestion
// #define PARANOID_TREE_CHECKS
```

Reply:
Good catch. Done!